hatchling-web-app/src/components/TextEditor.jsx

```diff
@@ -26,9 +26,34 @@ export default function Texteditor() {
     alert("A name was submitted: " + code);
   }
 
+  function onLoadButtonClick() {
+    //`current` points to the mounted file input element
+    console.log("Hello Wolrd")
+   inputFile.current.click();
+  }
+
+  function handelInputFile(event) {
+    var load_file = false;
+    const fileUploaded = event.target.files[0];
+
+    if (window.confirm('Are you sure you want to upload file \'' + fileUploaded.name + '\'?')) {
```

Comment:
```suggestion
    if (window.confirm(`Are you sure you want to upload file ${fileUploaded.name} ?`)) {
```

This line is better suited for string interpolation rather than multiple string concatenations.